### DISCUSSION TITLE
Not allowed to access end-to-end encrypted content

### DISCUSSION THREAD

**Discussion by vschwaberow** [nextcloud/all-in-one#4192] [Questions] [answered]
I have problems with end-to-end encryption. I have an user which is able to access all content but fails to delete or move content now. 

OCA\DAV\Connector\Sabre\Exception\Forbidden: Client "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36" is not allowed to access end-to-end encrypted content

Any idea how I can fix that?

---
In mobile agents "Write access to end-to-end encrypted folder requires token - no token sent" - the agent is corrected paired with the Nextcloud instance.

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question at https://help.nextcloud.com thanks!
